go-controller/pkg/metrics/master.go

```diff
@@ -156,6 +156,14 @@ var metricV6AllocatedHostSubnetCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The total number of v6 host subnets currently allocated",
 })
 
+var metricEgressFirewallRulesCount = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name: "num_egress_firewall_rules",
+	Help: "The total number of egress firewalls rules defined per namespace"},
```

Comment:
```suggestion
	Help: "The number of egress firewall rules defined per namespace"},
```
I'm not familiar with how this works, so forgive a naive reading:

`total number` and `per namespace` don't quite add up on first reading.

Seems to me you can have 
* a total number of firewall rules for ALL namespaces
* a total number of firewall rules for ONE namespace
* an AVERAGE number of firewall rules in all namespaces (this is what comes to mind when I read the current text with the `per namespace` language.

Again, I know NOTHING about this, just pointing out a potential point of confusion for users.